storage/tree_validation_test.go

```diff
@@ -87,6 +87,94 @@ func TestValidateTreeForCreation(t *testing.T) {
 	}
 }
 
+func TestValidateTreeForUpdate(t *testing.T) {
+	tests := []struct {
+		desc     string
+		updatefn func(*trillian.Tree)
+		wantErr  bool
+	}{
+		{
+			desc: "valid",
```

Comment:
`gofmt` didn't do anything. Why do you think it's hinky? (Apart from it looking like Go, I mean) ;)